podcasts/End of Year/Stories/YearOverYearStory.swift

```diff
@@ -74,29 +84,45 @@ struct YearOverYearStory: ShareableStory {
                 ZStack(alignment: .bottom) {
 
                     GeometryReader { proxy in
-                        HStack(alignment: .bottom, spacing: 0) {
-                            bar(
-                                title: "2022",
-                                subtitle: data.totalPlayedTimeLastYear.storyTimeDescription,
-                                geometry: geometry,
-                                barStyle: .grey
-                            )
-                            .frame(height: leftBarPercentageSize * proxy.size.height)
-
-                            bar(
-                                title: "2023",
-                                subtitle: data.totalPlayedTimeThisYear.storyTimeDescription,
-                                geometry: geometry,
-                                barStyle: .rainbow
-                            )
-                            .frame(height: rightBarPercentageSize * proxy.size.height)
-
+                        VStack(spacing: 0) {
```

Comment:
Added a `VStack` here so the animation goes from bottom to top. Without it, it goes top to bottom.